torch/csrc/autograd/input_buffer.cpp

```diff
@@ -5,6 +5,7 @@
 #include <ATen/TensorOperators.h>
 #include <ATen/TensorSubclassLikeUtils.h>
 #include <ATen/native/SparseTensorUtils.h>
+#include <torch/csrc/autograd/functions/accumulate_grad.h>
```

Comment:
This should not depend on accumulate grad.
If you need a shared util, you can create a utils file.